lib/plugins/filter/after_post_render/external_link.js

```diff
@@ -1,23 +1,35 @@
 'use strict';
 
 const { isExternalLink } = require('hexo-util');
+let EXTERNAL_LINK_POST_CONFIG;
+let EXTERNAL_LINK_POST_ENABLED = true;
 
 function externalLinkFilter(data) {
+  if (!EXTERNAL_LINK_POST_ENABLED) return;
+
   const { config } = this;
 
-  if (typeof config.external_link === 'undefined' || typeof config.external_link === 'object'
-    || config.external_link === true) {
-    config.external_link = Object.assign({
-      enable: true,
-      field: 'site',
-      exclude: []
-    }, config.external_link);
+  if (typeof EXTERNAL_LINK_POST_CONFIG === 'undefined') {
+    if (typeof config.external_link === 'undefined' || typeof config.external_link === 'object'
+      || config.external_link === true) {
+      EXTERNAL_LINK_POST_CONFIG = Object.assign({
+        enable: true,
+        field: 'site',
+        exclude: []
+      }, config.external_link);
+      // Write config back
+      config.external_link = EXTERNAL_LINK_POST_CONFIG;
```

Comment:
I'm personally ok with this, but user might caught off guard, https://github.com/hexojs/hexo-generator-feed/issues/115

Reply:
@curbengh Since the original code will write the config back to `config.external_link` directly, so I retain this approach in this PR.

However, I do agree to remove this line since it seems not break anything.